Google-Summer-of-Code-2024.md

```diff
@@ -51,7 +51,7 @@ Welcome! If you're interested in applying to work with Oppia for GSoC, please fo
 
 5. Select one or more [GSoC project ideas](#oppias-project-ideas-list) that you're most interested in, and write your project proposal! You can get feedback from project mentors when you've completed a sufficient draft -- see the instructions in the [GSoC proposal template](#gsoc-proposal-template) section for details.
 
-   We require that all general discussion about GSoC projects take place in open channels. If you have questions about a project, you can ask in [GitHub Discussions](https://github.com/oppia/oppia/discussions/categories/gsoc-2024-q-a). Please be specific when asking questions, since this makes it easier for us to help you.
+   We require that all general discussion about GSoC projects take place in open channels. If you have questions about a project, you can ask in [GitHub Web Discussions](https://github.com/oppia/oppia/discussions/categories/gsoc-2024-q-a) or [GitHub Oppia Discussions](https://github.com/oppia/oppia-android/discussions/categories/general-gsoc-q-a). Note that individual projects have their own categories, so please make use of those if you have project-specific questions. Please also be specific when asking questions, since this makes it easier for us to help you.
```

Comment:
```suggestion
   We require that all general discussion about GSoC projects take place in open channels. If you have questions about a project, you can ask in [GitHub Web Discussions](https://github.com/oppia/oppia/discussions/categories/gsoc-2024-q-a) or [GitHub Android Discussions](https://github.com/oppia/oppia-android/discussions/categories/general-gsoc-q-a). Note that individual projects have their own categories, so please make use of those if you have project-specific questions. Please also be specific when asking questions, since this makes it easier for us to help you.
```